Travel Scheduler/Views/CommentsCell.m

```diff
@@ -0,0 +1,94 @@
+//
+//  CommentsCell.m
+//  Travel Scheduler
+//
+//  Created by gilemos on 7/27/19.
+//  Copyright © 2019 aliu18. All rights reserved.
+//
+
+#import "CommentsCell.h"
+#import "Place.h"
+#import "UIImageView+AFNetworking.h"
+
+static UIImageView* makeImage(int width)
```

Comment:
Hehe, I was making this mistake the whole time too, but the asterisk should be next to the method name like "static UIImageView *makeImage(int width)"

Reply:
ooohh did not know! Gonna fix that, thanks!